caluma/extensions/events/form.py

```diff
@@ -102,11 +103,24 @@ def update_table_summary_from_table_question(instance, *args, **kwargs):
             update_table_summary(instance=ad)
 
 
+def get_translation_with_fallback(value):
+    lang = get_language()
+    fallback_langs = ["en", "de"]
+    fallback_langs.remove(lang)
+    translated = getattr(value, lang)
+    if not translated:
+        for fl in fallback_langs:
+            translated = getattr(value, fl)
+            if translated:
+                break
+    return translated
+
+
```

Comment:
Could probably be avoided by configuring localized fields, no? 

https://django-localized-fields.readthedocs.io/en/latest/settings.html#localized-fields-fallbacks

Reply:
This is [what we have already](https://github.com/adfinis/mySAGW/blob/main/caluma/extensions/settings.py#L57). But they just reported that this is not just a problem with the summaries, but with Analytics in general. I will have another look until continuing with this.

Reply:
Those are two different things. I've made issue [issue for it in Caluma](https://github.com/projectcaluma/caluma/issues/2333).

Somehow the `LOCALIZED_FIELDS_FALLBACKS` setting doesn't get picked up. We need this, so I'm gonna merge anyway.

Reply:
Fair enough, and yeah analytics might do things... differently :grimacing: